### ISSUE TITLE
Hot-key support for TimeControlNode

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#682]
TimeControlNode needs a comprehensive hot-key design.  It has some partical support, but there has been no holistic design.

One of its subcomponent (PlayPauseButton) supports a hot key by default, while the other subcomponents (StepForwardButton, StepBackwardButton) do not support hot keys.  PlayPauseButton hot key is enabled by default, which feels odd.  And MAL added its own StepForwardButton support, which should be in TimeControlNode, see https://github.com/phetsims/scenery-phet/issues/681.

This is not blocking for Fourier 1.0.  I'm planning to disabled hot-key support for TimeControlNode's subcomponents.

**Comment by pixelzoom** [comment]
Also consider adding common-code support for TimeControlNode keyboard help content.  See https://github.com/phetsims/fourier-making-waves/issues/93.

**Comment by pixelzoom** [comment]
In https://github.com/phetsims/fourier-making-waves/issues/93#issue-927708422, I noted that Fourier will (someday) need hotkey support for TimeControlNode. (This will not be included in Fourier 1.0, see https://github.com/phetsims/fourier-making-waves/issues/92#issuecomment-867799973.)

> **Discrete** screen:
> In #92, we'll eventually be adding hotkeys for TimeControlNode.  It current looks like the screenshot below, and uses SliderAndGeneralKeyboardHelpContent.  
> * Will there be common-code support for adding keyboard help content for TimeControlNode?  
> * Or will clients have to add (and duplicate) strings?  
> * What should the title be?

In https://github.com/phetsims/fourier-making-waves/issues/93#issuecomment-867057378, @jessegreenberg replied:

> The help content for the TimeControlNode hotkeys in MAL was
> ![tempsnip](https://user-images.githubusercontent.com/6396244/123147440-48c56180-d41c-11eb-8f03-09a17c04b065.png)
> 
> I think it would be great if we could have reusable strings for the TimeControlNode, but have been told we need unique strings for each sim. @arouinfar what do you think?
> 
> Even if we continue using unique strings we can make a common code component for the help content.
> 
> > Can I still use the SliderAndGeneralKeyboardHelpContent
> 
> Probably not, I am guessing we will create a new Node that extends TwoColumnKeyboardHelpContent, but lets see what the requested look of this dialog is for this sim first.

**Comment by pixelzoom** [comment]
Reviewing scenery-phet issues for Q4 2022 planning...

There has been no progress on this, assigned to @jessegreenberg .  We worked around this in Fourier, but it's needed in MOTHA. @kathy-phet please prioritize.

**Comment by zepumph** [comment]
Subset of https://github.com/phetsims/scenery/issues/1298

**Comment by jessegreenberg** [comment]
Reviewing this issue as of 2/23/24 - 

- `TimeControlNode` has support for a global hotkey to play/pause through `PlayStopButton`.
- There is now a `TimeControlKeyboardHelpSection` in scenery-phet with a heading and row for this command.
- We should design hot keys for TimeControlNode in general to determine if other keys should be added (such as for step forward/backward). Then we can update `TimeControlKeyboardHelpSection`.

**Comment by jessegreenberg** [comment]
We visited this during a prioritization meeting and want to have an approach for 'hotkey registry' working before we add more hotkeys to common code components like this.

**Comment by pixelzoom** [comment]
@jessegreenberg checking in on the status of this.  MOTHA supports alt input, there is a TimeControlNode in MOTHA, and I'm currently doing this in subclass MOTHATimeControlNode --  which is the same workaround that I used in FMW when this issue was created 3+ years ago.

```ts
        playPauseButtonOptions: {
          //TODO https://github.com/phetsims/scenery-phet/issues/682 workaround, we do not want partial hotkey support for TimeControlNode
          includeGlobalHotkey: false
        },
```

**Comment by zepumph** [comment]
I think we want to tag @samreid on this, who is currently working on dynamic layout and PhET-iO improvements in TimeControlNode.

**Comment by pixelzoom** [comment]
@samreid Could/should this issue be included in the scope of what you're doing elsewhere for TimeControlNode?

**Comment by arouinfar** [comment]
> And MAL added its own StepForwardButton support, which should be in TimeControlNode, see https://github.com/phetsims/scenery-phet/issues/681.

While MAL does have a hotkey related to StepForwardButton, it's nuanced. The hotkey steps in LARGER amounts than pressing the StepForwardButton. This was added to support the way that description operates in that sim, which depends on whether the sim is paused/stepped, playing at normal speed, or playing at slow speed.

If StepForwardButton were to have a hotkey in scenery-phet, it should behave like a single button press. The ALT+L hotkey in MAL behaves differently, so I do not think it should be generalized.

**Comment by samreid** [comment]
At today's PhET-iO meeting, @kathy-phet asked me to review recent changes to TimeControlNode that pertain to accessibility tier 1 with @jessegreenberg, so I'll bundle this in for that discussion.

**Comment by samreid** [comment]
@jessegreenberg I assigned parts of https://github.com/phetsims/scenery-phet/issues/826 to you for review. Can you please review this issue and recommend how to proceed as part of it?

**Comment by pixelzoom** [comment]
@DianaTavares please review, close if OK. The dialog is the same for both screens, see screenshot below.  As is the PhET convention, the sections are ordered by most-to-least specific, from left to right.  All of these sections are common-code, nothing sim-specific.

<img width="816" alt="Image" src="https://github.com/user-attachments/assets/5ad6713d-20eb-417d-9cb5-84d0fb3496c5" />
